### DISCUSSION TITLE
Windows 11 without Console Mode in next Release ?

### DISCUSSION THREAD

**Discussion by paule32** [magiblot/tvision#130] [General]
Hello,
I read a Internet Article about the new coming Windows 11 next year (don't ask me the about the URL - I don't remember me on this).
However, in the Article stands, that Microsoft plan is it, to remove the Console Mode for Windows 11, and future Releases.
Is that true or false ?
Because, when Windows has no Console Mode anymore, then we can't use all the Command Line Tools ?
What about this Project ?
I am a little bit confused in this, but what do you think about this Article ?
And: what does Microsoft mean with "Console Mode" ?

Happy Discussing
Jens

**Comment by ziotom78** [comment]
No, they are abandoning the old “legacy console mode”, which was used to run old programs refusing to work with Windows 10 Terminal app. I don't believe there is anything to worry: if your program works with the plain terminal in Windows 10, it will likely work in Windows 11 too.

P.S. I read about it here: <https://www.neowin.net/news/microsoft-deprecates-legacy-console-mode-in-windows-11-and-future-releases/>. Here is a more in-depth explanation: <https://4sysops.com/archives/switch-between-windows-terminal-and-the-legacy-console/>.

**Comment by magiblot** [comment]
Yeah, this is not a problem. Turbo Vision works with or without the legacy console.
